tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -158,24 +158,67 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         return pager;
     };
 
-    fluid.tests.tooltipModuleSource = function (pager, sequencesOfPageSizes) {
+    fluid.defaults("fluid.tests.pagerTooltipEnv", {
+        gradeNames: ["fluid.test.testEnvironment"],
+        markupFixture: "#rendered-ioc",
+        components: {
+            pager: {
+                type: "fluid.tests.renderedPager",
+                container: "#rendered-ioc",
+                options: {
+                    gradeNames: ["fluid.tests.tooltip.trackTooltips", "fluid.tests.focusNotifier"]
+                }
+            },
+            fixtures: {
+                type: "fluid.tests.pagerTooltipTests",
+                options: {
+                    sequenceOfPageSizes: [2, 3, 4]
```

Comment:
You have a default value of `[2, 3]` in the grade and then override it here. Is there a specific reason to do this, and not just put `[2, 3, 4]` in the grade definition?